recipes/rtmidi/all/patches/4.0.0-0003-use-rtmidi-folder.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 28ee1f8..37855fe 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -231,7 +231,7 @@ install(TARGETS ${LIB_TARGETS}
+         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+         ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+-        PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
++        PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/rtmidi)
```

Comment:
ok, but I think that include convention for 5.0.0 is broken: https://github.com/thestk/rtmidi/issues/285

You should set `includedirs` in `package_info()` to `os.path.join("include", "rtmidi")`

Reply:
Done here: 31a493f53629e29375ae5c471ec9b1e27ab998ac

Reply:
ok, you just have to update include in test package now